internal/log/log.go

```diff
@@ -8,42 +8,59 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-func init() {
+type Logger interface {
+	Info(msg string)
+	Debug(msg string)
+	Error(msg string, err error)
+	Fatal(msg string, err error)
+	Warn(msg string)
+	Request(msg string, r *http.Request)
+}
+
+type Log struct{}
+
+func NewLogger() *Log {
 	log.Debug().Msg("Logger started")
+	return &Log{}
 }
 
-func Info(msg string) {
+func (l Log) Info(msg string) {
 	log.Info().Msg(msg)
```

Comment:
Shouldn't these all be updated to `l` instead of `log`?

Reply:
Yep - gone ahead and updated this. The interface methods have the same names as the embedded zerolog package so its kind of verbose ie. logger.Logger.Info()